site/ja/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,3254 @@
+{
```

Comment:
<ol><li>Very weak suggestion: "同じサイズに設定しています。" is better than "同じサイズに設定します。" Because, to set buffer_size the same size as the dataset is good in this case but not in other cases with larger datasets.</li><li>In last paragraph, "を" are repeated. </li><li class="ql-indent-1">Suggestion: "tf.data.Dataset.apply メソッドを、...と組み合わせることで対処することができます。" </li></ol>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Good suggestions.  Thanks.</p>